gravitee-apim-integration-tests/src/test/java/io/gravitee/apim/integration/tests/http/pathparams/PathParametersV4IntegrationTest.java

```diff
@@ -36,7 +35,7 @@
  * @author GraviteeSource Team
  */
 @GatewayTest
-public class PathParametersV4IntegrationTest extends PathParametersIntegrationTest {
+public class PathParametersV4IntegrationTest extends PathParametersV3IntegrationTest {
```

Comment:
Should we precise V4 in the name of the class? (this remarks applies for all tests)
Should we now consider v4 as the de facto and so have:
```suggestion
public class PathParametersIntegrationTest extends PathParametersV3IntegrationTest {
```

Reply:
I always put V3 /V4 Emulation or V4 in the test just to be explicit.